14_functions_lab/01.py

```diff
@@ -1,7 +1,26 @@
-"""
-Write 2 functions:
-    mysum - returns the sum of its input arguments
-    mymul - returns the multiplication of its input arguments
-    Ignore non-numeric arguments
-"""
-
+"""
+Two fucntions: 1st is sum , 2nd is multiply.
+"""
+
+from types import *
+
+def sum(*args):
+    res = 0

```

Comment:
This works but has too much code duplication between sum and multiply. Can you reduce their duplicate code?